File: Scrapers/referenceaudio.py
from Scrapers.common import AshopCategoryScraper


class ReferenceAudioScraper(AshopCategoryScraper):
    def __init__(self):
        super().__init__(
            base_url="https://www.referenceaudio.se",
            name="Reference Audio",
            category_url="https://www.referenceaudio.se/kategori/935/begagnat",
        )

File: ts-scrapers/src/sites/referenceaudio.ts
import { AshopCategoryScraper } from '../bases/ashop.js';

export class ReferenceAudioScraper extends AshopCategoryScraper {
  constructor() {
    super({
      name: 'Reference Audio',
      baseUrl: 'https://www.referenceaudio.se',
      categoryUrl: 'https://www.referenceaudio.se/kategori/935/begagnat',
      slug: 'reference-audio',
    });
  }
}
